spec/models/cart_spec.rb

```diff
@@ -41,4 +41,36 @@
     end
   end
 
+  describe 'checkout' do
+    let(:user) {User.new(first_name: 'Cernan', last_name: 'Bernardo', email: 'cernan@flatironschool.com')}
```

Comment:
Here's an alternative to setting up your data 
```ruby
before(:each) do
  @cart = Cart.create(status: 'pending')
  @item = @cart.items.create(name: 'iPhone', inventory: 20, price: 200)
  @item2 = @cart.items.create(name: 'xbox', inventory: 10, price: 250
  @user = @cart.create_user(
    first_name: 'Cernan',
    last_name: "Bernardo",
    email: 'cernan@flatironschool.com')
end
```

That way you don't need separate blocks using let AND a `before(:each)`. You would then just need to use them as instance variables so you can reference them in each of your tests. 

```ruby
describe 'checkout' do
    before(:each) do
      @cart = Cart.create(status: 'pending')
      @item = @cart.items.create(name: 'iPhone', inventory: 20, price: 200)
      @item2 = @cart.items.create(name: 'xbox', inventory: 10, price: 250)
      @user = @cart.create_user(
        first_name: 'Cernan',
        last_name: "Bernardo",
        email: 'cernan@flatironschool.com')
    end

    it 'reduces the inventory for each item' do
      @cart.checkout

      expect(@item.inventory).to eq(19)
      expect(@item2.inventory).to eq(9)
    end

    it 'makes sure that the total spent by the user is updated' do
      @cart.checkout

      expect(@user.total_spent).to eq(450)
    end

    it 'changes the status of the cart to submitted' do
      @cart.checkout

      expect(@cart.status).to eq('submitted')
    end
  end
```